src/_posts/platform/getting-started/2000-01-01-troubleshooting-ssh.md

```diff
@@ -57,11 +57,14 @@ $ git push scalingo master
 ```
 
 If the branch named `master` does not exist (default branch may be named `main`) you need to create one:
-
 ```bash
-git checkout -b master
+$ git checkout -B master
 ```
 
+{% note %}
+  Scalingo doesn't support the `main` default branch currently. You need to use the `master` branch.
+{% endnote %}
```

Comment:
This is not exactly true, you can use a local `main` branch and push on Scalingo:

```
git push scalingo main:master
```